crypto/bio/bio_lib.c

```diff
@@ -348,15 +348,15 @@ static int bio_write_intern(BIO *b, const void *data, size_t dlen,
     return ret;
 }
 
-int BIO_write(BIO *b, const void *data, int dlen)
+int BIO_write(BIO *b, const void *data, size_t dlen)
```

Comment:
We can't change the signature of a public function.

Reply:
> We can't change the signature of a public function.

Sorry, I wasn't aware BIO_write is a public function - I will revert it...

Reply:
There are a number of other public functions you're modifying.  If a function is in `include/openssl/*.h`, it's considered public.

Reply:
> There are a number of other public functions you're modifying. If a function is in `include/openssl/*.h`, it's considered public.

I already noticed and reverted all of those. My last commit is identical to the first commit which does not change any `include/openssl/` headers. What I don't understand: My first commit resulted in a green CI while the latest one was read. Any idea why there are different results to the same code?